src/gpb_compile.erl

```diff
@@ -3242,54 +3242,14 @@ opt_specs() ->
       "       instead of -include, which is the default.\n"},
      {"type_specs_hrl", undefined, type_specs_hrl, "\n"
      "        Exports definitions to hrl file."},
-     {"type", undefined, type_specs, "\n"
```

Comment:
Can you please explain why this file has been changed?

Reply:
A mistake from merging - these options were duplicated and occur elsewhere in this list.

Reply:
Do you mean, you made some mistakes while merging and here you have fixed these errors?

Reply:
Yes - I didn't look too closely when merging this section and duplicated some of the options.